server/routes/auth.js

```diff
@@ -1,6 +1,29 @@
 import { Router } from "express";
-import { Pool } from "pg";
+import pool from '../db.js'
+import passport from "passport";
+import bcrypt from "bcrypt";
+
 
 const router = Router();
 
+
+
+router.post("/login",passport.authenticate('local'), (req, res)=>{
+    res.json(req.session.passport.user)
+})
+
+router.post("/logout", (req, res)=>{
+    req.logout();
+    res.status(401).send({msg:"log out successfully"})
```

Comment:
I would say 401 is incorrect here as the logout has been successful in my mind it would be a `200` in addition, it is often good when someone calls logout to redirect the user to the login page again

Reply:
sure I will change it to 200.
Im thinking the redirection can occur from the frontend.I can send the success msg so frontend can redirect it to login page.